cypress/integration/CancerTypeKeywordSearch/cancerTypeSubtypeStageSideEffectTest.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types="Cypress" />
+
+import { Given, Then, And, When } from "cypress-cucumber-preprocessor/steps";
+
+Then("field with placeholder appears", dataTable => {
```

Comment:
This is old version of the code that still uses the forAttr from the table - the Map is missing